db_menager.py

```diff
@@ -0,0 +1,17 @@
+import mysql.connector
+import os
+
+
+class DB_Menager():
```

Comment:
DBManager 
(for classes use CamelCase, for variable snake_case)

and rename mEnager to Manager

Reply:
![image](https://user-images.githubusercontent.com/54208127/203144444-999b057e-b349-4a93-8372-dda4d3ac266b.png)